### PR TITLE
Navigation header redesign

### PR DESCRIPTION
**Pull request by ddouglasz**
#### Summary

Navigation header redesign

Expanded:
image

Collapsed:
image

### CODE REVIEW

File: packages/application-shell/src/components/navbar/messages.ts [hunk 1]
```diff
@@ -5,4 +5,8 @@ export default defineMessages({
     id: 'NavBar.MCSupport.title',
     defaultMessage: 'Support',
   },
+  navBarHeader: {
+    id: 'NavBar.navBarHeader',
+    defaultMessage: 'Merchant Center',
+  },
```
Comment (emmenko) [inline packages/application-shell/src/components/navbar/messages.ts:11]: There is no need to define this as a message, the term `Merchant Center` doesn't need a translation. You can hardcode it in the component.

**Review by emmenko — COMMENTED** [review]
Looking at the screenshots, it seems to me that the logo is somehow misaligned 🧐
Can you double check?

**Comment by emmenko** [comment]
PS: a changeset is missing

File: packages/application-shell/src/components/navbar/messages.ts (hunk 1 above)
Comment (ddouglasz) [reply packages/application-shell/src/components/navbar/messages.ts:11]: Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3216/commits/a19545223dd01fad077d7c4de15e2c64222c5b0c

Thank you.

Comment (kark) [reply packages/application-shell/src/components/navbar/messages.ts:11]: I guess we could clean up this message now 🙂

File: packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css [hunk 2]
```diff
@@ -25,6 +25,30 @@
 
 .navigation-header {
```
Comment (kark) [inline packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css:26]: It still looks slightly more centered in our designs 
| Designs |  Implementation | 
|---|---|
|  Screenshot 2023-09-15 at 11 37 31 |  Screenshot 2023-09-15 at 11 37 46 |

**Review by kark — COMMENTED** [review]
Thank you Douglas for working on this 🙌

File: packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css (hunk 2 above)
Comment (ddouglasz) [reply packages/application-shell/src/components/navbar/navbar-new/navbar.mod.css:26]: Oh yes I noticed the misalignment,
Thinking about it now, the content should be centered and not aligned with the contents on the side nav. 
Thank you

Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3216/commits/40c59028af074317acb832b4b353589510cecfe6

**Review by kark — APPROVED** [review]
Thank you!

**Pull request merged**